pyeparse/__init__.py

```diff
@@ -4,8 +4,8 @@
 
 from . import utils  # noqa
 from .edf._raw import RawEDF  # noqa
-Raw = RawEDF  # noqa
```

Comment:
This will be a backward-incompatible change, but I suppose it's okay since AFAIK only @dengemann and people from my lab at UW use this